static/js/components/tiles/top_event_tile.tsx

```diff
@@ -323,4 +310,22 @@ export function TopEventTile(props: TopEventTilePropType): JSX.Element {
   function formatDateString(dateString: string): string {
     return dateString.slice(0, "YYYY-MM-DD".length);
   }
+
+  function addEventLink(eventId: string, displayStr: string): JSX.Element {
+    return <a href={`/browser/${eventId}`}>{displayStr}</a>;
+  }
+
+  function getDisplayDate(event: DisasterEventPoint): string {
+    let ret = "";
+    if (props.topEventMetadata.showStartDate && event.startDate) {
+      ret += formatDateString(event.startDate);
+      if (props.topEventMetadata.showEndDate && event.endDate) {
+        ret += " - ";
```

Comment:
Should we use something besides " - " because it could look a little confusing since the date strings use "-" as well? so it might become something like "2020-01-01-2022-01-01"

Reply:
> Should we use something besides " - " because it could look a little confusing since the date strings use "-" as well (could end up with something like "2020-01-01-2022-01-01")

Reply:
changed to an emdash. wanted something visually simpler
![image](https://user-images.githubusercontent.com/6052978/218576302-85ef8abc-715a-41c8-8b31-49c8d6189051.png)

Reply:
nice, emdash looks good!